orc8r/cloud/Makefile

```diff
@@ -102,7 +104,7 @@ gen: tools $(GEN_LIST)
 $(GEN_LIST): %_gen:
 	make -C $*/cloud/go gen
 
-fullgen: clean_gen gen swagger tidy
+fullgen: clean_gen swagger_directories gen swagger tidy
```

Comment:
swagger_directories actually is a requirement of gen now, so move it to `gen: tools swagger_directories $(GEN_LIST)`